### PR TITLE
Uskis Arable Land - missed quest

### PR DESCRIPTION
**Pull request by vtange**

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -3317,33 +3317,33 @@ QUEST_LV_0200_20150323_003316	$When I thought about the moment when this scriptu
...
+QUEST_LV_0200_20150323_003325	$Even during Medis Diena, this place was safe.{nl}I used to think disasters were somebody else's problem.
+QUEST_LV_0200_20150323_003326	$The bees are not the only ones that changed.{nl}The monsters that I saw for the first time in my life had overrun Siauliai Forest back then.
+QUEST_LV_0200_20150323_003327	$I've spent many years making mead, but I have to end now without any significant product.{nl}Who should I blame for this..
 QUEST_LV_0200_20150323_003328	{memo X}All other Revelators were wrong.{nl}But if it were you.. I don't know.
-QUEST_LV_0200_20150323_003329	The priests told us to wait, and the damages that were caused by the monsters were bad.{nl}I could not stand anymore so I stood up myself.
-QUEST_LV_0200_20150323_003330	I will find out why monsters are making mess here.{nl}Then the priests will not say anything.
+QUEST_LV_0200_20150323_003329	The priest told us to wait, but the damage caused by the monsters was getting bad.{nl}I could not stand it anymore so I took action.
+QUEST_LV_0200_20150323_003330	I will find out why the monsters are making a mess here.{nl}Then the priest will be speechless.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:3330]: Priest -> priests 

(Cant check the korean now, but if I remember correctly, it said 'priests' for these quests)

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -3317,33 +3317,33 @@ QUEST_LV_0200_20150323_003316	$When I thought about the moment when this scriptu
...
-QUEST_LV_0200_20150323_003327	I spent many years to make the honey alcohol, but I have to end now without any siginificant result.{nl}Who should I blame for this..
+QUEST_LV_0200_20150323_003325	$Even during Medis Diena, this place was safe.{nl}I used to think disasters were somebody else's problem.
+QUEST_LV_0200_20150323_003326	$The bees are not the only ones that changed.{nl}The monsters that I saw for the first time in my life had overrun Siauliai Forest back then.
+QUEST_LV_0200_20150323_003327	$I've spent many years making mead, but I have to end now without any significant product.{nl}Who should I blame for this..
 QUEST_LV_0200_20150323_003328	{memo X}All other Revelators were wrong.{nl}But if it were you.. I don't know.
-QUEST_LV_0200_20150323_003329	The priests told us to wait, and the damages that were caused by the monsters were bad.{nl}I could not stand anymore so I stood up myself.
-QUEST_LV_0200_20150323_003330	I will find out why monsters are making mess here.{nl}Then the priests will not say anything.
+QUEST_LV_0200_20150323_003329	The priest told us to wait, but the damage caused by the monsters was getting bad.{nl}I could not stand it anymore so I took action.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:3329]: Priest -> priests

File: QUEST_LV_0200.tsv [hunk 3]
```diff
@@ -3317,33 +3317,33 @@ QUEST_LV_0200_20150323_003316	$When I thought about the moment when this scriptu
...
 QUEST_LV_0200_20150323_003332	$The priest is very angry.{nl}Damn, if only she told me more about the altar.
 QUEST_LV_0200_20150323_003333	$The villagers told us that they couldn't trust the priest anymore so they were taking action themselves.{nl}Me? I don't care, as long as the monsters get taken care of.
 QUEST_LV_0200_20150323_003334	$Where could Goddess Austeja have gone at a time like this?
-QUEST_LV_0200_20150323_003335	I know that the priests try their best, but we should protect our village.{nl}We have a trouble living on due to those monsters so we can't just do nothing.
-QUEST_LV_0200_20150323_003336	The priests never tell us what's wrong.{nl}We know that they are afraid that we would feel unstable.. But still we want to know.
+QUEST_LV_0200_20150323_003335	I know that the priest is trying her best, but we should protect our village.{nl}We're having trouble living on due to those monsters, so we can't just do nothing.
+QUEST_LV_0200_20150323_003336	The priest never told us what's wrong.{nl}We know that she is afraid that we might feel uneasy.. But still we want to know.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:3336]: Priest -> priests
She is -> they are

File: QUEST_LV_0200.tsv [hunk 4]
```diff
@@ -3317,33 +3317,33 @@ QUEST_LV_0200_20150323_003316	$When I thought about the moment when this scriptu
...
+QUEST_LV_0200_20150323_003330	I will find out why the monsters are making a mess here.{nl}Then the priest will be speechless.
 QUEST_LV_0200_20150323_003331	$Maybe we touched something we shouldn't have touched.{nl}I think we did..
 QUEST_LV_0200_20150323_003332	$The priest is very angry.{nl}Damn, if only she told me more about the altar.
 QUEST_LV_0200_20150323_003333	$The villagers told us that they couldn't trust the priest anymore so they were taking action themselves.{nl}Me? I don't care, as long as the monsters get taken care of.
 QUEST_LV_0200_20150323_003334	$Where could Goddess Austeja have gone at a time like this?
-QUEST_LV_0200_20150323_003335	I know that the priests try their best, but we should protect our village.{nl}We have a trouble living on due to those monsters so we can't just do nothing.
-QUEST_LV_0200_20150323_003336	The priests never tell us what's wrong.{nl}We know that they are afraid that we would feel unstable.. But still we want to know.
+QUEST_LV_0200_20150323_003335	I know that the priest is trying her best, but we should protect our village.{nl}We're having trouble living on due to those monsters, so we can't just do nothing.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:3335]: Priest is trying her -> priests are trying their

File: QUEST_LV_0200.tsv [hunk 5]
```diff
@@ -3317,33 +3317,33 @@ QUEST_LV_0200_20150323_003316	$When I thought about the moment when this scriptu
...
 QUEST_LV_0200_20150323_003337	$I know this might come out wrong but, I only trust you alone, Revelator.{nl}I am sorry to the priests.. and I am sorry to you, Revelator, for saying that.
-QUEST_LV_0200_20150323_003338	So we created the incident.{nl}Because of us.. I am so sorry.{nl}
+QUEST_LV_0200_20150323_003338	So we created the incident.{nl}Our actions.. I am so sorry.{nl}
 QUEST_LV_0200_20150323_003339	$I'm scared Goddess Austeja won't come back to protect our village..{nl}But, I am even more scared of the monsters in front of my eyes.
 QUEST_LV_0200_20150323_003340	$I refuse to just wait around as the priests have said.{nl}I refuse to be driven around by those monsters anymore.
 QUEST_LV_0200_20150323_003341	$I don't want to doubt the Goddesses.{nl}But the priests don't tell us anything.. Isn't that strange?
-QUEST_LV_0200_20150323_003342	There's so much distrust between our priests and the villagers.{nl}I don't want to create a mess..{nl}
+QUEST_LV_0200_20150323_003342	There's so much distrust between our priest and the villagers.{nl}I don't want to create a mess..{nl}
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:3342]: Priest -> priests

**Comment by vtange** [comment]
Good to Merge

**Comment by Sunaries** [comment]
Good to merge. 👍

**Comment by imcgames** [comment]
:+1:

**Pull request merged**